app/build.gradle

```diff
@@ -42,9 +42,9 @@ dependencies {
     implementation 'com.android.support:design:27.1.1'
 
     implementation 'com.jakewharton.timber:timber:4.7.1'
+    implementation 'com.squareup.tape2:tape:2.0.0-beta1'
```

Comment:
Can we not use generic methods to save files rather than using an external library?

Reply:
Generic Methods?

Reply:
Are you referring to making our own SaveData class and using it ?

Reply:
Yes. Methods written by us. Is there any specific reason to use this library?

Reply:
How will we save it @CloudyPadmal? How will we serialize it to disk? If we spend time on writing serializing logic rather than using well-defined libraries like GSON/Moshi, we'll lose time we can spend on making this app

Reply:
Ahh now I get the point :+1: